packages/ui/src/domainModules/taxon/components/TaxonNameTaxonStatus/index.js

```diff
@@ -5,46 +5,45 @@ import { connect } from 'react-redux'
 import { Link } from 'react-router-dom'
 import objectPath from 'object-path'
 
-import { createGetNestedItemById } from 'coreModules/crud/higherOrderComponents'
+import { createGetItemById } from 'coreModules/crud/higherOrderComponents'
 import { withI18n } from 'coreModules/i18n/higherOrderComponents'
 
 const mapStateToProps = (state, { formValueSelector }) => {
   return {
+    acceptedToTaxonId: formValueSelector(state, 'acceptedToTaxon.id'),
```

Comment:
no, I said select it from the crud module nestedItems cache. you might have to change back the rest in this file after you fix that.